Sources/DeepLearning/Layer.swift

```diff
@@ -209,6 +263,13 @@ public struct Dense<Scalar: TensorFlowFloatingPoint>: Layer {
         self.activation = activation
     }
 
+    /// Returns the output obtained from applying the layer to the given input.
+    ///
+    /// - Parameters
```

Comment:
```suggestion
    /// - Parameters:
```

Reply:
Thanks for the catch. These are copied from the `applied(to:)` protocol requirement comments, which are not from this patch.